app/src/components/domain/GoMultiFileInput/index.tsx

```diff
@@ -97,6 +97,7 @@ function GoMultiFileInput<T extends NameType>(props: Props<T>) {
         formData: true,
         url,
         method: 'POST',
+        useCurrentLanguageForMutation: true,
```

Comment:
We should pass this down as prop and use it only where necessary, this might affect the other places where input is currently being used